code/modules/admin/verbs/debug.dm

```diff
@@ -974,6 +974,8 @@ GLOBAL_PROTECT(AdminProcCallSpamPrevention)
 		if(!zlevel_turf_indexes["[T.z]"])
 			zlevel_turf_indexes["[T.z]"] = list()
 		zlevel_turf_indexes["[T.z]"] |= T
+		T.color = "red"
+		T.maptext = "ACT_TURF"
```

Comment:
Bad idea, we use this on the live server.

I wouldn't mind a "Mark ATs in world (do not use on live)" question though

Reply:
hide it under a different set of verbs so the admin monkeys can't use it

Reply:
I can just handle this client side then

Reply:
handled client side now